hbase-shell/src/main/ruby/shell.rb

```diff
@@ -108,6 +108,8 @@ class Shell
     # exit the interactive shell and save that this
     # happend via a call to exit
     def exit(ret = 0)
+      # Non-deamon Netty threadpool in ZK ClientCnxnSocketNetty cannot be shut down otherwise
+      self.hbase.shutdown
```

Comment:
If `shutdown` throws an exception then shell won't exit. Shouldn't we ignore the exceptions?

Reply:
Makes sense.
Modified to ignore exceptions from close.